app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionPanelView.kt

```diff
@@ -120,19 +132,64 @@ class TrackingProtectionPanelView(
 
     private fun setUIForDetailsMode(
         category: TrackingProtectionCategory,
-        categoryBlocked: Boolean
+        categoryBlocked: Boolean,
+        categorySmartBlocked: Boolean
     ) {
         normal_mode.visibility = View.GONE
         details_mode.visibility = View.VISIBLE
         category_title.setText(category.title)
+
         blocking_text_list.text = bucketedTrackers.get(category, categoryBlocked).joinToString("\n")
+
         category_description.setText(category.description)
         details_blocking_header.setText(if (categoryBlocked) {
             R.string.enhanced_tracking_protection_blocked
         } else {
             R.string.enhanced_tracking_protection_allowed
         })
 
+        // show description for SmartBlock tracking content in details
+        if (category == TRACKING_CONTENT) {
```

Comment:
I think we can have content that was unblock in any category  [see](https://github.com/mozilla-mobile/android-components/pull/10676/files#diff-332e50b45d5cf7dda87dd76b059e54ac5c9050b9c7af6cac021ef143249c5f7dR22)